Ginger/Ginger/ALM/Repository/QtestRepository.cs

```diff
@@ -298,6 +298,13 @@ public override bool ExportActivitiesGroupToALM(ActivitiesGroup activtiesGroup,
             //TODO: retireve test case fields -->DONE
             ObservableList<ExternalItemFieldBase> testCaseFields =  WorkSpace.Instance.Solution.ExternalItemsFields;
             var filterTestCaseFields = testCaseFields.Where(tc => tc.ItemType == eQCItemType.TestCase.ToString()).ToList();
+            for(var i = 0; i < activtiesGroup.ActivitiesIdentifiers.Where(ai => ai.IdentifiedActivity == null).ToList().Count; i++)
```

Comment:
For loop is taking count from where conition, and inside loop its modifying i of list without where conition. please check it